Tools/ardupilotwaf/boards.py

```diff
@@ -369,6 +369,17 @@ def configure_env(self, cfg, env):
         if cfg.options.ekf_single:
             env.CXXFLAGS += ['-DHAL_WITH_EKF_DOUBLE=0']
 
+        # add files from ROMFS_custom
+        custom_dir = 'ROMFS_custom'
+        if os.path.exists(custom_dir):
+            for root, subdirs, files in os.walk(custom_dir):
+                for f in files:
+                    if fnmatch.fnmatch(f,"*~"):
```

Comment:
If we're adding exceptions, it's bound to get out of hand. Why not use a specific extension? Although I guess the "custom files" part names that complicated.. but still.

Reply:
it's just a tidyness thing. The lua scripting looks for files ending in ".lua" anyway

Reply:
and I wanted to be able to put in config files for the scripts